src/chunk.c

```diff
@@ -2422,9 +2425,11 @@ ts_chunk_get_window(int32 dimension_id, int64 point, int count, MemoryContext mc
 		for (j = 0; j < ccs->num_constraints; j++)
 		{
 			ChunkConstraint *cc = &ccs->constraints[j];
-			Chunk *chunk = ts_chunk_get_by_id(cc->fd.chunk_id, true);
+			Chunk *chunk = ts_chunk_get_by_id(cc->fd.chunk_id, false);
```

Comment:
Could you elaborate why it is okay to not  find a chunk here ?
(The commit message is also a bit confusing. It says: 
```
Ignore chunks marked as dropped and recreate the
Data Node list when resurrecting any chunks with tombstones.
````
Where are we ignoring chunks that are dropped here? For the resurrect function, we are resurrecting them.

Reply:
We are ignoring the chunks in line 2431 of `ts_chunk_get_window()`. The resurrection is a different case than `ts_chunk_get_window()`.

It is ok to not find a chunk as shown in line 2444:
```
	/* Assert that we never return dropped chunks */
```
It's just that the code was wrong before.